server/parsec/templates/account_email_deletion.html

```diff
@@ -0,0 +1,555 @@
+
+<body>
+  <!-- These lines are used to isolate the preheader and avoid a preview of the mail's content -->
+  <span class="preheader"
+    style="color: transparent; display: none; height: 0; max-height: 0; max-width: 0; mso-hide: all; visibility: hidden; width: 0;">
+    Please confirm your account deletion
+  </span>
+
+  <table role="presentation" border="0" cellpadding="0" cellspacing="0" class="main-table"
+    style="background-color: #fff; width: 100%;">
+    <tr>
+      <td align="center">
+        <table cellspacing="0" cellpadding="0" width="600" class="card-table">
+          <!-- Header Section -->
+          <tr class="header-section">
+            <td>
+              <table cellspacing="0" cellpadding="0" width="100%">
+                <tr class="logo">
+                  <td>
+                    <img src="{{ server_url }}/static/logo-parsec-row.png" alt="Parsec Logo" title="Parsec" width="128"
+                      style="height: 21.9px;" />
+                  </td>
+                </tr>
+                <tr class="title">
+                  <td>
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    <p style="line-height: 28.9px; color: #51517B; font-size: 14px;">
+                    </p>
+                  </td>
+                </tr>
+              </table>
+            </td>
+          </tr>
+
+          <!-- Main Body Content Placeholder -->
+          <tr class="body-section">
+            <td>
+              <table cellspacing="0" cellpadding="0" width="100%">
+                <tr>
+                  <td class="content-wrapper">
+                    <table cellspacing="0" cellpadding="0" align="center" width="100%"
+                      style="border-collapse: collapse;">
+                      <tr class="content-section first-section">
+                        <td class="info-frame">
+                          <table class="section-content" role="presentation" border="0" cellpadding="0" cellspacing="0">
+                            <tr>
+                              <td>
+                                <table>
+                                  <tr class="content-number">
+                                    <td class="content-number-circle">
+                                      <p class="content-number-item">1</p>
+                                    </td>
+                                  </tr>
+                                  <tr style="float: left; display: block;">
+                                    <td>
+                                      <p class="content-title">Download Parsec application</p>
+                                    </td>
+                                  </tr>
+                                  <tr>
+                                    <td style="padding-top: 16px;">
+                                      <p class="content-section-text">
+                                        Make sure to download and install Parsec:</p>
```

Comment:
Comparing it with the txt template, it seems that a phrase is missing before the actual steps: "To confirm your Parsec Account deletion:"

Reply:
I've added the 'to confirm ...' in the title in the previous block

Reply:
https://github.com/Scille/parsec-cloud/pull/10480#issuecomment-2949219182